.github/workflows/bump-version-flake.yml

```diff
@@ -0,0 +1,48 @@
+name: Bump nixpacks-flake version
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  AWK_CMD: '{sub(/${{ env.REGEX }}/, "v0.0.21"); print}'
+  FLAKE_FILE: flake.nix
+  GIT_AUTHOR_NAME: ${{ github.repository_owner }}
+  GIT_AUTHOR_EMAIL: ${{ github.repository_owner }}@users.noreply.github.com
+  REGEX: v[0-9]+\.[0-9]+\.[0-9]+
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get the release version from the tag and old version
+        shell: bash
+        if: env.NIXPACKS_VERSION == ''
+        run: |
+          echo ${}
+          echo "NIXPACKS_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "version is: ${{ env.NIXPACKS_VERSION }}"
+
+      - uses: actions/checkout@v3
+        with:
+          ref: auto-update/nixpacks-flake
```

Comment:
This branch needs to already exist right? I think it would be better if
- The main branch was checked out
- New branch created off main (name based off version)
- Version in flake is updated to version of tag
- Commit flake file to new branch
- Create pull request

Reply:
I thought taht branch was created on checkout 😕, but yes, that's the ideal flow

Reply:
I don't have much experience writing gh actions. checking out a branch inside a step and then commit some changes will really take effect?

Reply:
I am pretty sure it should yes. Do you have a test repo or something to test on? We can try some things out there.

Reply:
no, but I can create one for sure. I'll do it right now